ambry-frontend/src/main/java/com/github/ambry/frontend/AccountAndContainerInjector.java

```diff
@@ -44,6 +45,7 @@ public class AccountAndContainerInjector {
   private static final Set<String> requiredAmbryHeadersForPutWithAccountAndContainerName = Collections.unmodifiableSet(
       new HashSet<>(Arrays.asList(Headers.TARGET_ACCOUNT_NAME, Headers.TARGET_CONTAINER_NAME)));
   private static final Logger logger = LoggerFactory.getLogger(AccountAndContainerInjector.class);
+  private static final String PATH_SEPARATOR_STRING = "/";
```

Comment:
might as well just put this in the RestUtils so it can used by almost all the package.

Reply:
Sure.